src/Mixin.js

```diff
@@ -82,8 +82,13 @@ export const BaseMixin = sessionGetter => ({
     update(this, this.props, this.state);
   },
 
+  componentDidMount() {
+    this._isMounted = true;
```

Comment:
Since the component that uses this mixin might already be using `_isMounted` for its own tracking, how about instead we set `this._rethinkMixinState.isMounted`? We're already using that instance variable on line 80 so it can't possibly cause further issues.